### DISCUSSION TITLE
Fixed: Organization-owned Repo doesn't show non-Org Sponsor.

### DISCUSSION THREAD

**Discussion by LlmDl** [community/community#5774] [Other Feature Feedback, Questions, & Ideas]
My repo is Organization-owned but I use my personal Sponsor badge in the funding.yml. At some point in the last few days the sponsor button at the top of the repo has started to point to the Organization's main page when clicked:
![Capture](https://user-images.githubusercontent.com/879756/134554372-ee367a46-5bad-4bea-824b-84e716810b9f.PNG)

There has also been an added box at the bottom of the Issues tickets' pages which, when clicked, prompts me to sign up the Organization to the Sponsors program.
![Capture2](https://user-images.githubusercontent.com/879756/134554406-d5b603e3-95a9-42b1-9876-9c8a06debd1f.PNG)

Neither button nor issue-page-box point to my sponsor information in the funding.yml.

The only place where there is a sponsorship button that works is on the repo's home-page sidebar.

It would be great that the funding.yml would be in control of what the header sponsor button and the bottom of the issues pages did when clicked on.

**Comment by jlord** [comment]
Thank you for reporting this! I'll open a ticket with our Engineering team 👍

**Reply by jlord** [reply]
Realized engineering has already been on it! Should be fixed now :D

**Reply by LlmDl** [reply]
Looks like the button at the top of the repo (where the unwatch button is) has been fixed.

The banner at the bottom of issue tickets is not using the funding.yml still, thanks for responding @jlord, can you ask them if they still have something in the works for that?

**Reply by LlmDl** [reply]
Looks like the bottom button has been fixed as well. Thank you Engineering team!

**Reply by jlord** [reply]
Woot! 🎉
